fill-column-indicator.el

```diff
@@ -616,8 +620,11 @@ rough heuristic.)"
                    (dolist (win (fci-get-buffer-windows t))
                      (when (display-images-p (window-frame win))
                        (throw 'found-graphic (window-frame win))))))))
-    (setq fci-char-width (frame-char-width frame)
-          fci-char-height (frame-char-height frame))))
+    (setq fci-char-width (window-font-width)
+          fci-char-height (window-font-height)
+          fci-scaled-column (round (* (float (or fci-rule-column fill-column))
```

Comment:
Why use the `(or fci-rule-column fill-column)` here instead of the `fci-column` defined earlier?

Could the scaling code be moved into a helper (internal) function so the rounding is not duplicated?